pandas/tests/test_panel.py

```diff
@@ -1560,6 +1560,17 @@ def test_truncate_fillna_bug(self):
         # it works!
         result.fillna(value=0.0)
 
+    def test_truncate_outofbounds(self):
```

Comment:
The latest (force) push uses ```assert_series_equal``` below. Were you pointing out something different you would like to see?